contracts/core/EntryPoint.sol

```diff
@@ -506,8 +513,9 @@ contract EntryPoint is IEntryPoint, StakeManager, NonceManager, ReentrancyGuard,
                 revert FailedOp(opIndex, "AA31 paymaster deposit too low");
             }
             paymasterInfo.deposit = deposit - requiredPreFund;
+            uint256 pmVerificationGasLimit = mUserOp.paymasterVerificationGasLimit;
```

Comment:
I'm curious why this variable makes things more efficient than accessing `mUserOp.paymasterVerificationGasLimit` directly below.

Reply:
I use it twice in this method. so it saves (a little..) to keep the memory var on the stack.